.circleci/config.yml

```diff
@@ -173,7 +173,8 @@ jobs:
       - run:
           name: Setup Python environment
           command: |
-            pyenv global 3.5.2
+            pyenv install 3.6.3
+            pyenv global 3.6.3
```

Comment:
If we don't need to build docker images, then we can just do:

```diff
   test_pytest:
-    machine:
-      image: circleci/classic:201711-01
+    docker:
+      - image: circleci/python:latest
     working_directory: /home/circleci/src/smriprep
```

And skip the `pyenv`.